src/util.cpp

```diff
@@ -697,7 +697,43 @@ boost::filesystem::path GetTempPath() {
 
 void runCommand(std::string strCommand)
 {
-    int nErr = ::system(strCommand.c_str());
+    int nErr = 0;
+#ifdef WIN32
+    // If we're in Windows, use CreateProcess.
+    ULONG nExitCode;
+    STARTUPINFO info={sizeof(info)};
+    ZeroMemory(&info, sizeof(STARTUPINFO));
+    info.cb = sizeof(STARTUPINFO);
+    info.dwFlags = STARTF_USESHOWWINDOW;
+    info.wShowWindow = SW_HIDE;
+    PROCESS_INFORMATION processInfo;
+    ZeroMemory(&processInfo, sizeof(PROCESS_INFORMATION));
+    char system32[256];
+    if (GetSystemDirectory(system32, 255)) {
+        std::stringstream ssCommand;
+        ssCommand << system32 << "\\cmd.exe /C " << strCommand;
+        std::string strTempCmd = ssCommand.str();
+        std::vector<char> vecCommand(strTempCmd.begin(), strTempCmd.end() + 1);
```

Comment:
OTOH I don't see anywhere in the [CreateProcess](https://msdn.microsoft.com/en-us/library/windows/desktop/ms682425%28v=vs.85%29.aspx) documentation that it will actually try to change the passed-in string. So this should suffice:
```
std::string strTempCmd = std::string(system32) + "\\cmd.exe /C " + strCommand;
if (CreateProcess(NULL, const_cast<char*>(strTempCmd.c_str()), NULL, NULL, false, 0, NULL, NULL, &info, &processInfo))
```